.gitignore

```diff
@@ -22,8 +22,6 @@ packages/**/yalc.lock
 ehthumbs.db
 Thumbs.db
 
-e2e-projects/**/.next
```

Comment:
why do we need this if the folder has been completely removed from the project?

Reply:
Hum don't get it. It's actually removed. Maybe you just need to delete the folder once for all

Reply:
damn, sorry, for some reason ealier i thought this was a +, lol